src/pages/monitor/backup-and-restore.html

```diff
@@ -0,0 +1,74 @@
+<div>
+    <title>${messages.app.title} : ${messages.jmx.QueryMonitoringPage.title}</title>
+
+    <link href="css/backup-and-restore.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
+
+    <h1>
+        {{title}}
+        <span class="btn btn-link"
+              uib-popover-template="'js/angular/templates/titlePopoverTemplate.html'"
+              popover-trigger="mouseenter"
+              popover-placement="bottom-right"
+              popover-append-to-body="true">
+            <span class="icon-info text-tertiary"></span>
+        </span>
+        <span onto-loader-new class="backup-and-restore-updater-icon" size="25">
```

Comment:
There is one loader here and one at the end of the page. Is this the same way as other pages' heading?

Reply:
Ok, perhaps we should create an issue to update all other pages that show information which is constantly updated.